setup.py

```diff
@@ -92,7 +93,7 @@
         ],
     },
     classifiers=[
-        "Development Status :: 5 - Production/Stable",
+        "Development Status :: 3 - Alpha",
```

Comment:
Doesn't this change the development status for all Snowpark Python API?

Reply:
fixed